test/README.md

```diff
@@ -0,0 +1,21 @@
+# Multi-Language Integration Test Runner
+This directory contains the source code for a fully-functional language-agnostic "integration test runner".
```

Comment:
I'd add a bit more about the overall intent of this module to make it really clear why it exists. Something like: 
> This test runner is used by the SDK code examples team to automate integration testing of all code examples across all supported languages.